simtools/visualization/visualize.py

```diff
@@ -544,7 +544,7 @@ def plot_hist_2D(data, **kwargs):
     return fig
 
 
-@u.quantity_input()
+@u.quantity_input(x=u.m, y=u.m, radius=u.m, equivalencies=[(u.cm, u.km)])
```

Comment:
As Gernot requested accepting other units of length, I included accepting also cm and km. but this was just because I couldnt find a type "Length" and accept all units of length, so this was the fast solution I encountered. if you know a simpler way to do it we could update this line

Reply:
I thought that if someone provides `cm`, then astropy would automatically convert it to `m` even without the `equivalencies` option. I thought `equivalencies` is needed in case the conversion isn't straightforward, like in the case of wavelength, frequency and energy. Did you try it without the `equivalencies` option?

Reply:
That is good to know. it works without it, so I am removing it. Thanks for pointing it out